HelpfulItem.h

```diff
@@ -1,15 +1,11 @@
 #pragma once
 
 #include "Item.h"
-#include "Dwarf.h"
 #include "Character.h"
-#include "Paladin.h"
-#include "DragonSlayer.h"
-#include "Dragon.h"
+#include "Utility.h"
```

Comment:
Are you using anything from this `Utility.h` in this header file? 
If not, then don't include it.
Don't Include what you Don't Use.
Alternatively, Only Include what you do use.
- [ ] revise